lib/puppet/provider/package/puppetserver_gem.rb

```diff
@@ -1,4 +1,8 @@
+require 'facter'
+require 'hocon'
```

Comment:
Ah, one further thing. The `puppet-agent` package does include a build of the hocon gem, but not all Puppet agents might have it. Core Puppet does have a `hocon` feature that can be used to avoid a load error in these cases. That would be a swap of the `require` for the following in the provider body:

```ruby
confine :feature => :hocon
```

Reply:
Done, thanks!